docs/AOTA.md

```diff
@@ -0,0 +1,43 @@
+# AOTA Updates
+
+## Supported AOTA commands and their functionality:
+
+### Supported ***docker*** commands
+
+| *docker* Command | Definition                                                                                                                                                                                                                                       |
+|:-----------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Import           | Importing an image to the device and starting a container                                                                                                                                                                                        |
+| List             | Lists all containers for all images that are either 'latest' or have a tag number.  It will list the container ID, state, and image name.  It will provide 'NONE' for the container ID and state if the image does not have an active container. | 
+| Load             | Loading an image from the device and starting a container                                                                                                                                                                                        |
```

Comment:
If I remember correctly the image doesn't necessary need to be on the device. Image always comes from a URL (which could be hosted locally).

Reply:
I changed the wording to how the docker documentation phrases the docker load command.